sdk/messaging/azservicebus/internal/namespace.go

```diff
@@ -37,7 +34,7 @@ type (
 	Namespace struct {
 		Name          string
 		Suffix        string
-		TokenProvider auth.TokenProvider
+		TokenProvider *tokenProvider
```

Comment:
This also should not be exported then?

Reply:
Haha, independently came to the same conclusion. Fixed.

Reply:
Ah, oops, I focused on the wrong thing there.

TokenProvider is used in some tests external from the internal package.